lib/sengiri/model/base.rb

```diff
@@ -66,16 +66,32 @@ def sharding_group(name, confs: nil, suffix: nil)
 
         def dbconfs
           if defined? Rails
-            @dbconfs ||= Rails.application.config.database_configuration.select {|name|
-              /^#{@sharding_group_name}/ =~ name
-            }.select {|name|
-              /#{env}#{@sharding_database_suffix}$/ =~ name
-            }
-
+            @dbconfs ||= rails_db_confs
           end
           @dbconfs
         end
 
+        def rails_db_confs
```

Comment:
🆓 In the method name, I do not a bit clearly understand the difference between `dbconfs` and `rails_db_confs`.

Since A does not cache, how about giving a prefix such as `parse_`?